src/ModelTraits/SpatieTranslatable/HasTranslations.php

```diff
@@ -65,10 +66,10 @@ public static function create(array $attributes = [])
             if ($model->isTranslatableAttribute($attribute)) { // the attribute is translatable
                 $model->setTranslation($attribute, $locale, $value);
             } else { // the attribute is NOT translatable
-                $model->{$attribute} = $value;
+                $non_translatable[$attribute] = $value;
             }
         }
-        $model->save();
+        $model->fill($non_translatable)->save();
```

Comment:
Aye, I agree that the upstream code doesn't check for the result of the save either - ergo the replacement is, really, no worse.

On the other hand, if the original `save()` or the newer `fill` followed by the `save` doesn't work, there's still something wrong...

I wonder if:

```
if (! $model->fill($non_translatable)->save()) {
  Log::error("When attempting to save the save failed.");
  // 1. This assumes the Log façade is used.
 // 2. The message is seriously wonky and should be improved.
}
```

The above example is, as the comments suggest, a bit *wonky* but I've honestly stared at code wondering WHY something wasn't saving or such simply because some other code *assumed* that a save would never fail (:imp:). 

We could consider logging that something went wrong; it's still opaque to the end user but there's more chance that something will be noticed vs someone not noticing that the save itself could fail rather silently.